.github/workflows/release.yml

```diff
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
```

Comment:
Is there a reason not to update these also in the 'Build' workflow?

Reply:
thought you're updating it in the caching PR. will add